components/Fields/Field.vue

```diff
@@ -94,6 +96,9 @@ export default defineNuxtComponent({
     shortDescription(): string | undefined {
       return this.properties?.description?.replace(/(<([^>]+)>)/gi, '')
     },
+    isWebsite(): boolean {
+      return /\w*:?website:?\w*/.test(this.field.field)
```

Comment:
This regex match too many unwanted possibilities.

We just need `this.field.field === 'website' || this.field.field.startsWith('website:') || this.field.field.endsWith(':website')`

Reply:
Not interested on such case (and for such cache `.*website.*` is ok). The ':' are important here.

Reply:
we could constraint the regex more if you want ?
Instead of `\w` we could specify [a-ZA-Z_1] ?
something like this looks more precise ? 
```
^[a-zA-Z_1]*:?website:?[a-zA-Z_1]*$
```

Reply:
The equivalent regex is `(\w+:)?website(:\w+)?`.  If you use `:?` alone you make it optional. If you use `[..]*` you also make it optional and allow eg `:website:`.